### PR TITLE
openjdk@11: update to 11.0.2.

### PR DESCRIPTION
**Pull request by yxwangcs**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

The current link in `openjdk@11` formula is broken, this pull request fixes the link and bumps the version to `11.0.2`.

### CODE REVIEW

File: Formula/openjdk@11.rb [hunk 1]
```diff
@@ -3,16 +3,12 @@ class OpenjdkAT11 < Formula
...
   if OS.mac?
     url "https://java.com/"
   else
-    url "https://download.java.net/java/early_access/jdk11/28/GPL/openjdk-11+28_linux-x64_bin.tar.gz"
-    sha256 "3784cfc4670f0d4c5482604c7c513beb1a92b005f569df9bf100e8bef6610f2e"
-  end
-
-  bottle do
-    sha256 "2866dc00248063e5736c5675271a2f65e7ab187202d4e217eb3317f280a7474d" => :x86_64_linux
+    url "https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x64_bin.tar.gz"
```
Comment (iMichka) [inline Formula/openjdk@11.rb]: Please leave the bottle block (without the sha256)

**Comment by iMichka** [comment]
The bottle blocks is now gone for openjdk@11. Please rebase this on master, and just update the url to the upstream binary.

**Comment by yxwangcs** [comment]
It is now rebased and updated. Thanks!

**Comment by iMichka** [comment]
Looks like something went wrong with the rebase. There are now 1261 commits in this pull request.

Can you have a look and try to clean this up. If not, you can also close this pull request and open a fresh one.

**Pull request closed without merging**